include/RAJA/index/IndexSetUtils.hpp

```diff
@@ -90,11 +84,12 @@ RAJA_INLINE void getIndicesConditional(CONTAINER_T& con,
                                        CONDITIONAL conditional)
 {
   CONTAINER_T tcon;
-  forall<ExecPolicy<seq_segit, seq_exec> >(iset,
-    [&](typename CONTAINER_T::value_type idx) {
-      if (conditional(idx)) tcon.push_back(idx);
-    }
-  );
+  forall<ExecPolicy<seq_segit, seq_exec>>(
+      iset,
```

Comment:
why is it indenting 4 spaces here and 2 spaces in other places?

Reply:
it's applying an argument alignment rule instead of an indentation rule